content/_index.md

```diff
@@ -1,7 +1,16 @@
 ---
 title: "Home"
 seo_title: "Jakarta® EE | Cloud Native Enterprise Java | Java EE | The Eclipse Foundation"
-headline: "<img class='jumbotron-title' src='/images/jakarta/jakarta-ee-text.svg' alt='Jakarta EE'><br>BUILDING AN OPEN SOURCE ECOSYSTEM FOR CLOUD NATIVE ENTERPRISE JAVA"
+headline: |
+  <img class='jumbotron-title' src='/images/jakarta/jakarta-ee-text.svg' alt='Jakarta EE'>
+  <br>
+subtitle: Building an open source ecosystem for <div></div>cloud native architectures with Enterprise Java
```

Comment:
The div here is a little odd, is that intentionally there? I'd have thought of a BR over a block component here

Reply:
It is super odd, and I wanted to use &nbsp; instead, but it wasn't working. 

A <br> has semantic meaning and wouldn't be suitable here. The line break here is more stylistic.

An alternative would be to add a span with a class to set it as `display: block`. The class could be named "line-break". It might be more obvious for someone reading the file what the intended purpose is... but it is technically the same.

Reply:
Actually I'll just use `aria-hidden` or `role=presentation` on a `br`